app/models/user.rb

```diff
@@ -9,9 +9,7 @@ class User < ApplicationRecord
   belongs_to :local_school, optional: true
   belongs_to :course, optional: true
   has_many :charges
-  has_many :course_registrations
-  has_many :courses, through: :course_registrations
-
+  belongs_to :courses
```

Comment:
Don't other models need to be updated? Make sure all these associations are working